### ISSUE TITLE
Problem generating JSON file

### ISSUE THREAD

**Issue by jalakoo** [espresso3389/flutter_oss_licenses#13]
Generating the default .dart file works just fine, but get this error when attempting to output JSON.

```
% flutter pub run flutter_oss_licenses:generate.dart -o licenses.json --json
Converting object to an encodable object failed: Instance of 'Package': #0      _JsonStringifier.writeObject (dart:convert/json.dart:794:7)
#1      _JsonPrettyPrintMixin.writeList (dart:convert/json.dart:899:7)
#2      _JsonStringifier.writeJsonValue (dart:convert/json.dart:824:7)
#3      _JsonStringifier.writeObject (dart:convert/json.dart:785:9)
#4      _JsonStringStringifier.printOn (dart:convert/json.dart:983:17)
#5      _JsonStringStringifier.stringify (dart:convert/json.dart:968:5)
#6      JsonEncoder.convert (dart:convert/json.dart:345:30)
#7      main (file:///Users/Shared/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_oss_licenses-2.0.0+1/bin/generate.dart:39:51)
<asynchronous suspension>
```

Encountering this on a mac running:
Flutter 2.10.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 5f105a6ca7 (3 months ago) • 2022-02-01 14:15:42 -0800
Engine • revision 776efd2034
Tools • Dart 2.16.0 • DevTools 2.9.2

**Comment by Wissperwind** [comment]
Same Problem here on Windows.

PS C:\test\test\test\webgui\test> flutter pub run flutter_oss_licenses:generate.dart -o licenses.json --json
Converting object to an encodable object failed: Instance of 'Package': #0      _JsonStringifier.writeObject (dart:convert/json.dart:794:7)
#1      _JsonPrettyPrintMixin.writeList (dart:convert/json.dart:899:7)
#2      _JsonStringifier.writeJsonValue (dart:convert/json.dart:824:7)
#3      _JsonStringifier.writeObject (dart:convert/json.dart:785:9)
#4      _JsonStringStringifier.printOn (dart:convert/json.dart:983:17)
#5      _JsonStringStringifier.stringify (dart:convert/json.dart:968:5)
#6      JsonEncoder.convert (dart:convert/json.dart:345:30)
#7      main (file:///C:/Users/username/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_oss_licenses-2.0.0+1/bin/generate.dart:39:51)
<asynchronous suspension>

**Comment by lmerten** [comment]
Same Problem:

```
Converting object to an encodable object failed: Instance of 'Package': 
#0      _JsonStringifier.writeObject (dart:convert/json.dart:794:7)
#1      _JsonPrettyPrintMixin.writeList (dart:convert/json.dart:899:7)
#2      _JsonStringifier.writeJsonValue (dart:convert/json.dart:824:7)
#3      _JsonStringifier.writeObject (dart:convert/json.dart:785:9)
#4      _JsonStringStringifier.printOn (dart:convert/json.dart:983:17)
#5      _JsonStringStringifier.stringify (dart:convert/json.dart:968:5)
#6      JsonEncoder.convert (dart:convert/json.dart:345:30)
#7      main (file:///Users/user/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_oss_licenses-2.0.0+1/bin/generate.dart:39:51)
```

Flutter 3.0.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision ee4e09cce0 (10 days ago) • 2022-05-09 16:45:18 -0700
Engine • revision d1b9a6938a
Tools • Dart 2.17.0 • DevTools 2.12.2

Doctor summary:
[✓] Flutter (Channel stable, 3.0.0, on macOS 12.3.1 21E258 darwin-x64)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
[✓] Xcode - develop for iOS and macOS (Xcode 13.4)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.0)
[✓] IntelliJ IDEA Ultimate Edition (version 2022.1)
[✓] VS Code (version 1.61.0)
[✓] Connected device (3 available)
[✓] HTTP Host Availability

**Comment by espresso3389** [comment]
Released as [flutter_oss_licenses  2.0.1](https://pub.dev/packages/flutter_oss_licenses/versions/2.0.1)/[dart_pubspec_licenses 2.0.2](https://pub.dev/packages/dart_pubspec_licenses/versions/2.0.2).
